test/alias_dialog_test.rb

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env rspec
+
+require_relative "test_helper"
+
+require "yast"
+
+class AliasDialog
+  include Yast::I18n
+  include Yast::UIShortcuts
+
+  def initialize
+    Yast.include self, "network/lan/virtual.rb"
+  end
+end
+
+describe "VirtualEditDialog" do
+  # bnc#864264
+  it "passes smoke test" do
+    include Yast::UIShortcuts
+
+    Yast.import "UI"
+    Yast.import "LanItems"
+
+    allow(Yast::UI).to receive(:UserInput).and_return(:ok)
+    allow(Yast::UI)
+      .to receive(:QueryWidget)
+      .and_return("")
+    allow(Yast::UI)
+      .to receive(:QueryWidget)
+      .with(Id(:ipaddr), :Value)
+      .and_return("1.1.1.1")
+    allow(Yast::UI)
+      .to receive(:QueryWidget)
+      .with(Id(:netmask), :Value)
+      .and_return("255.0.0.0")
+    allow(Yast::LanItems).to receive(:device).and_return("")
+
+    expect(Yast::UI).to receive(:UserInput).once
+    expect(AliasDialog.new.VirtualEditDialog(0, term(:empty), nil)).not_to be nil
```

Comment:
`0, term(:empty), nil` ? Even if this is only a smoke test, I have absolutely no idea what the arguments mean. I'll write an improvement later today.